app/main.jsx

```diff
@@ -25,9 +26,10 @@ const ExampleApp = connect(
 render(
   <Provider store={store}>
     <Router history={browserHistory}>
-      <Route path="/" component={Products}>
-        <IndexRedirect to="/products" />
+      <Route path="/navbar" component={Navbar} />
```

Comment:
We're working on an AppContainer right now in a new branch/attached to a new card. We built the /navbar route just for temporary testing purposes, and our new branch reflects a layout that is similar to Juke.